src/content/protocols/compound-v2.md

```diff
@@ -0,0 +1,227 @@
+
+Compound v2 currently relies on a non-upgradeable oracle contract to deliver price feeds via Chainlink. While the contract itself cannot be upgraded, its configuration is governed by the Compound community multisig with broad administrative powers: it can change the Chainlink price feed source, add or remove asset markets, or set a fixed fallback price for individual tokens. This power on the price feed could be abused and lead to _loss of user funds_ through unlunjustified liquidations.
+
+Finally, the governance itself is upgradeable: `GovernorBravoDelegator` proxy points to a delegate contract whose admin may be changed. COMP holders propose and vote on code changes, but once passed, any proposal can swiftly reassign all protocol-level privileges.
+
+> Upgradeability score: High
+
+## Autonomy
+
+Compound v2 relies on Chainlink price feeds to price collateral and base assets in the system. The protocol does not validate asset prices returned by Chainlink. The only fallback in place is the possibility for the governance to define default fixed prices for assets that will be used in case the oracle fails.
+
+`PiceOracle` fetches prices from Chainlink Price Feeds when requested for a specific cToken.
+
+The Chainlink oracle system itself is upgradeable without decentralized ownership over those permissions as we analyzed in a dedicated review [here](/protocols/chainlink-oracles). This dependency thus introduces centralization risk in the Compound v2 protocol.
+
+> Autonomy score: High
+
+## Exit Window
+
+Permissions, including protocol upgrades, are controlled by an onchain governance system. `COMP` holders are able to create new proposals (requires 25,000 `COMP`) and vote on proposals (at least 400,000 votes are required for a valid proposal). A minimum voting period of 3 days is enforced as well as a delay of 2 days for the implementation of successful proposals.
+
+While this does not meet the 7-day exit window requirement, malicious or unintended proposals can be intercepted by the `ProposalGuardian` multisig account.
+
+However, the [ProposalGuardian](#security-council) multisig do not meet the Security Council requirements ([see below](#security-council)).
+
+> Exit Window score: High
+
+## Accessibility
+
+The original Compound V2 interface ( [app.compound.finance](https://app.compound.finance/?market=usdc-mainnet) ) is fully open-source; anyone can clone the repo, swap the public Graph endpoint (or run their own sub-graph) and launch a local or IPFS-hosted copy in a few minutes. In practice, users are not limited to this front-end: Compound V2 positions can be opened, closed, or managed from a wide list of third-party dashboards and wallets (DeFi Saver, Instadapp, Zapper, Zerion, Debank, and more). Because there are multiple independent ways to reach the contracts even if the official website goes offline, user funds remain accessible.
```

Comment:
```suggestion
The original Compound V2 interface [app.compound.finance](https://app.compound.finance/?market=usdc-mainnet) is fully open-source; anyone can clone the repo, swap the public Graph endpoint (or run their own sub-graph) and launch a local or IPFS-hosted copy in a few minutes.

In practice, users are not limited to this front-end: Compound V2 positions can be opened, closed, or managed from a wide list of third-party dashboards and wallets (DeFi Saver, Instadapp, Zapper, Zerion, Debank, and more). 

Since there are multiple independent ways to reach the contracts even if the official website goes offline, user funds remain accessible.
```